tests/unit/CMakeLists.txt

```diff
@@ -1,6 +1,24 @@
 set(TEST_NAME deepworks_unit_tests)
 
 enable_testing()
+
+include(FetchContent)
```

Comment:
I believe this download stuff can be moved to `thirdparty`, and tests only link `pytorch`